Makefile

```diff
@@ -25,3 +25,23 @@ coverage:
 coveralls: coverage
 	go get github.com/mattn/goveralls
 	goveralls -coverprofile=$(COVERAGE_FILE) -service=github
+
+.PHONY: bench
+bench:
+	go clean -testcache
+	go test -bench=. -benchmem -benchtime=1s
+
+.PHONY: ab-perf
+ab-perf:
+	export TNT_CUR_BRANCH=$(git branch --show-current)
+	go clean -testcache
+	go test -bench=. -benchmem -benchtime=1s > b.txt
+	git stash && git fetch && git checkout master
+	go clean -testcache
+	go test -bench=. -benchmem -benchtime=1s > a.txt
+	go get golang.org/x/tools/cmd/benchcmp
```

Comment:
> Deprecated: benchcmp is deprecated in favor of benchstat: golang.org/x/perf/cmd/benchstat

From https://pkg.go.dev/golang.org/x/tools/cmd/benchcmp.

Reply:
Thank you, fixed.